utils/user-product-helper.ts

```diff
@@ -52,3 +67,27 @@ export function getProductInfo (config, type: ProductCode): ProductI {
       } as ProductI
   }
 }
+
+/**
+ * Check if products array has MHR and PPR
+ */
+export function hasMhrAndPprProducts(products: Array<APIProductI>): boolean {
+  return products
+  .filter(product => product.code === ProductCode.MHR || product.code === ProductCode.PPR)
+  .length === 2
+}
+
+/**
+ * Filter our MHR and PPR products and add one MHR_PPR product instead (i.e. My Asset Registries)
+ */
+export function combineAssetProducts(products: Array<APIProductI>): Array<APIProductI>{
+  const combinedProducts = products.filter(product =>
+    product.code !== ProductCode.MHR && product.code !== ProductCode.PPR)
+
+  combinedProducts.push({
+    code: ProductCode.PPR_MHR,
+    subscriptionStatus: ProductStatus.ACTIVE,
+  } as APIProductI)
+
+  return combinedProducts
+}
```

Comment:
I like the approach but is it required to create a unique code explicit to having both roles?
Would a simple computed value that checks for both roles suffice?

I feel like you could avoid having to add a new code and the creation of it. 

something like..
`['ppr', 'mhr'].every(role => { return products.includes(role) })`

Reply:
This unique `productCode` is used to get content for the tile from the `user-product-helper`.

`Products` is an array of objects that has a `ProductCode` in them. We need to iterate thru each product and check for the code. 

<img width="400" alt="Screen Shot 2022-05-31 at 8 50 15 AM" src="https://user-images.githubusercontent.com/2333290/171216043-f2b6480c-ebc4-48e5-8bc4-1dc9c3685f3a.png">

Reply:
Ok fair enough, if you feel this is the best way i'm all for it 👍